File: 167. Two Sum II - Input Array Is Sorted/167. Two Sum II - Input Array Is Sorted.java
    class Solution {
  public int[] twoSum(int[] numbers, int target) {
    
    int left=0,right=numbers.length-1;
    
    while(left<right)
    {
        int sum=numbers[left]+numbers[right];
        if(sum==target)
            return new int[]{left+1,right+1};
        else if(sum>target)
            right--;
        else if(sum<target)
            left++;
    }
    return new int[]{};
}

File: 167. Two Sum II - Input Array Is Sorted/167. Two Sum II - Input Array Is Sorted.cpp
class Solution
{
public:
    vector<int> twoSum(vector<int> &nums, int target)
    {
        vector<int> ans;                // Answer Array
        int i = 0, j = nums.size() - 1; // Initilize Two Pointers
        while (i < j)
        {
            if (nums[i] + nums[j] < target)
                ++i;
            else if (nums[i] + nums[j] > target)
                --j;
            else
            {
                ans.push_back(i + 1);
                ans.push_back(j + 1);
                break;
                ++i;
            }
        }
        return ans;
    }
};